src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml

```diff
@@ -0,0 +1,62 @@
+setLeader: Test your knowledge of pseudo classes
+height: unset
+tabLabel: question
+questions:
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "0"
+    stem: |
+      Pseudo-classes select elements based on internal browser state, while
```

Comment:
I am not a fan of this question. I had to read it several times to understand what you were asking. Can we phrase this more simply?

Pseudo-classes act as if a class has been dynamically applied to an element, while pseudo-elements act on an element itself,